src/App.jsx

```diff
@@ -1,33 +1,30 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+// import { useState } from 'react'
 import './App.css'
 
+const todoList = [];
+[
+  'Master React', 
+  'Master Node', 
+  'Clear out email'
+].forEach((title, id) => { todoList.push({id, title}) });
+
```

Comment:
Another way of achieving this is using map, when using map you're implementing immutability in your code, which generally is good. Programs with immutable objects are less complicated to think about, since you don't need to worry about how an object may evolve over time.

https://medium.com/@redsage00/the-art-of-immutability-62e6f45f4525

```suggestion
const todoList =  [
  'Master React', 
  'Master Node', 
  'Clear out email'
].map((title, id) => { todoList.push({id, title}) });
```